graphql/user/fields.ts

```diff
@@ -210,6 +210,18 @@ export class UserFieldsResolver {
         return await getAppointmentsForUser(user, take, skip, cursor, direction);
     }
 
+    @FieldResolver((returns) => [Achievement])
+    @Authorized(Role.ADMIN, Role.OWNER)
+    async achievement(@Ctx() context: GraphQLContext, @Arg('id') id: number): Promise<Achievement> {
+        const achievement = await getAchievementById(context.user, id);
+        return achievement;
+    }
+    @FieldResolver((returns) => [Achievement])
+    @Authorized(Role.ADMIN, Role.OWNER)
+    async nextStepAchievements(@Ctx() context: GraphQLContext): Promise<Achievement[]> {
```

Comment:
Maybe rename?
```suggestion
    async achievementsNextSteps(@Ctx() context: GraphQLContext): Promise<Achievement[]> {
```

Reply:
Well this actually is not really the next step of an achievement. The sequential achievements get diesplayed as next step card on the front page.
So if you really would prefer another name we should go for sth like achievementsAsNextSteps. You have any more ideas?